lib/rspecq/worker.rb

```diff
@@ -69,6 +76,8 @@ def work
         # to `requeue_lost_job` inside the work loop
         update_heartbeat
 
+        break if @queue.fail_fast_limit_reached?
```

Comment:
Repeating a previous comment: can't this be moved together with `return if job.nil? && queue.exhausted?`?